.github/workflows/build-pr.yml

```diff
@@ -131,14 +131,13 @@ jobs:
       - name: Az CLI login
         uses: azure/login@v1
         with:
-          client-id: ${{ env.AZURE_CLIENT_ID }}
-          tenant-id: ${{ env.AZURE_TENANT_ID }}
-          subscription-id: ${{ env.AZURE_SUBSCRIPTION_ID }}
+          creds: '{"clientId":"${{ env.AZURE_CLIENT_ID }}","clientSecret":"${{ secrets.AZURE_CLIENT_SECRET }}","subscriptionId":"${{ env.AZURE_SUBSCRIPTION_ID }}","tenantId":"${{ env.AZURE_TENANT_ID }}"}'
```

Comment:
The access token lifetime of OIDC based login is limited to 5 min, which will expire in the test. Logging with SP directly can extend the token lifetime to 1 hour to unblock it.

Reply:
Or would this still be a problem since the `AZURE_CLIENT_SECRET` field must exist in the fork too then?

Reply:
> Or would this still be a problem since the `AZURE_CLIENT_SECRET` field must exist in the fork too then?

It will not fix the id-token write access issue. :( Actually I tried this way before as well.